hivemind/modules/api/tests/models/test_article.py

```diff
@@ -0,0 +1,25 @@
+"""
+Tests for API Module Signals
+"""
+
+from django.test import TestCase
+
+from hivemind.modules.api.models.article import Article
+
+
+class TestArticle(TestCase):
+    """
+    Test for Article Model
+    """
+
+    def test_article_str_representation(self):
+        """
+        Expects that the headline is returned.
+        """
+
+        article = Article(
```

Comment:
In order to simplify creation and management of Objects we have a package called [model-mommy](https://model-mommy.readthedocs.io/en/latest/basic_usage.html),

Reply:
The `UserProfile` model test has an example of its usage. I think it would be better to use model-mommy in tests such as this one.